CustomText.Utilities.CognitiveSearch/Solution/CustomTextSkillsetAzureFunction/CustomTextSkillsetAzureFunction.csproj

```diff
@@ -9,7 +9,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="CustomText.MiniSDK">
-      <HintPath>..\..\Release\CustomTextMiniSDKClient.dll</HintPath>
+      <HintPath>..\CustomTextMiniSDK\CustomText.MiniSDK.Client\bin\Debug\netcoreapp3.1\CustomText.MiniSDK.dll</HintPath>
```

Comment:
why did you change this?

Reply:
It was missing from the dependencies

Reply:
i specifically recall pushing this .dll file
*.dll is ignored by default (as specified in the .gitignore file) 

can you please place this file in the "release" folder and update your reference?